### PR TITLE
new package py-testpath

### PR DESCRIPTION
**Pull request by healther**
dependency of py-nbconvert

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-testpath/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    homepage = "https://github.com/jupyter/testpath"
+    url      = "https://pypi.io/packages/source/t/testpath/testpath-0.4.2.tar.gz"
+
+    version('0.4.2', sha256='b694b3d9288dbd81685c5d2e7140b81365d46c29f5db4bc659de5aa6b98780f8')
+
+    depends_on('py-pip', type='build')
+
+    def install(self, spec, prefix):
+        pip = which('pip')
+        pip('install', '.', '--prefix={}'.format(prefix))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-testpath/package.py]: Is this install method and pip dependency actually needed? Pip is usually only needed to install wheels.

Comment (healther) [reply var/spack/repos/builtin/packages/py-testpath/package.py]: It failed for me without, but I didn't investigate further, so there might be an option do go without it

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-testpath/package.py]: Post the error message and I’ll take a look.

Comment (healther) [reply var/spack/repos/builtin/packages/py-testpath/package.py]: couldn't reproduce.... That makes me wonder what kind of strange setup I had... This OS upgrade was significantly more painful than the last one.

**Pull request merged**